figs/paper/paper.Rmd

```diff
@@ -0,0 +1,101 @@
+
+# Underlying Principles
+
+Regardless of the hypothesis test in question, an analyst asks the same kind of question when conducting statistical inference: is the effect/difference in the observed data real, or due to random chance? To answer this question, the analyst begins by assuming that the effect in the observed data was simply due to random chance, and calls this assumption the *null hypothesis*. (In reality, they might not believe in the null hypothesis at all---the null hypothesis is in opposition to the *alternate hypothesis*, which supposes that the effect present in the observed data is actually due to the fact that "something is going on.") The analyst then calculates a *test statistic* from the data that describes the observed effect. They can use this test statistic to calculate a *p-value* via juxtaposition with a *null distribution*, giving the probability that the observed data could come about if the null hypothesis were true. If this probability is below some pre-defined *significance level* $\alpha$, then the analyst can reject the null hypothesis.
+
+The workflow of this package is designed around this idea. Starting out with some dataset,
+
++ `specify()` allows the analyst to specify the variable, or relationship between variables, that they are interested in.
++ `hypothesize()` allows the analyst to declare the null hypothesis.
++ `generate()` allows the analyst to generate data reflecting the null hypothesis.
++ `calculate()` allows the analyst to calculate summary statistics, either from
+     * the observed data, to form the observed test statistic.
+     * data `generate()`d to reflect the null hypothesis, to form the randomization-based null distribution of test statistics.
+
+As such, the ultimate output of an infer pipeline using these four functions is generally an _observed statistic_ or _null distribution_ of test statistics. These four functions are thus supplemented with many utilities to visualize and extract value from their outputs.
+
++ `visualize()` plots the null distribution of test statistics.
+     * `shade_p_value()` situates the observed statistic in the null distribution, shading the region as or more extreme.
+     * `shade_confidence_interval()` situates the confidence interval region in the null distribution, shading the region within the bounds.
++ `get_p_value()` calculates a p-value via the juxtaposition of the test statistic and null distribution.
++ `get_confidence_interval()` calculates a confidence interval via the juxtaposition of the test statistic and null distribution.
+
```

Comment:
So I think what's above is a bit confusing because most of the narrative is about hypothesis testing but then there are some bits scattered around for constructing confidence intervals.

I would suggest:

- Remove the bullet point about `shade_confidence_interval()` under `visualize()`
- Remove the bullet point about `get_confidence_interval()`
- Add another paragraph that says something along the lines of 

> The workflow outlined above can also be used for constructing confidence intervals via bootstrapping with the omission of the `hypothesize()` step in the pipeline. The resulting bootstrapping distribution can then be visualized with `visualize()`, the confidence interval region can be overlaid on this distribution using shading with `shade_confidence_interval()`, and the bounds of the confidence interval can be calculated with `get_confidence_interval()`.

I think this works as a paragraph but you could also make it bullet points if you like. It's important that we don't refer to a null distribution when talking about the confidence interval so I think this clarifies matters by referring to it as the bootstrapping distribution.